client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -112,8 +112,10 @@ export const getTotalPriceBreakdown = ( state, orderId, siteId = getSelectedSite
 	} : null;
 };
 
-const getAddressErrors = ( { values, isNormalized, normalized, selectNormalized, ignoreValidation }, countriesData ) => {
-	if ( isNormalized && ! normalized ) {
+const getAddressErrors = ( { values, isNormalized, normalized, selectNormalized, ignoreValidation, fieldErrors }, countriesData ) => {
+	if ( isNormalized && ! normalized && fieldErrors ) {
+		return fieldErrors;
+	} else if ( isNormalized && ! normalized ) {
```

Comment:
I know you're inheriting these prop names, but could we rename them to be more clear in their meaning? At a glance I can't see the difference between `isNormalized` and `normalized`.

Reply:
👍 Renamed `normalized` to `normalizedValues` locally — can do it everywhere if that's what you had in mind.

Reply:
That's good for now - gives us context [for later](https://github.com/Automattic/woocommerce-services/issues/1345).